### DISCUSSION TITLE
Multiple package_name variables in Appfile

### DISCUSSION THREAD

**Discussion by dusanristic** [fastlane/fastlane#21984] [General] [answered]
I have defined multiple flavors in my Android app. Therefore, I have mulitple apps with different package names.
Can I define all those packages in the Appfile, and then choose different ones for connecting to the Google Play Console depending on the lane? If yes, how can I do that?

**Comment by dusanristic** [comment, marked as answer]
I managed to solve it like:

```
package_name("A") # e.g. com.krausefx.app


for_lane :release do
    package_name('B')
end
```

**Comment by Amjd36913** [comment]
package_name("A") # e.g. com.krausefx.app


for_lane :release do
    package_name('B')
end

**Comment by coalacorey** [comment]
What I did and works quite well (since we have a lot of flavors and targets) is to create .env files for each flavor where I define the package name, development team, etc. and then in lane buildFlavorA I simply call Dotenv.load ".env.flavorA".

You can see all (environment) variables for a action by executing "fastlane action [action]" (e.g. firebase_app_distribution) and set these values accordingly in the .env.flavor file.
